ecpy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -453,6 +484,68 @@ def to_send(self, name, waveform):
         self._driver.write_binary_values(header, waveform, datatype='B')
         self.write('*WAI')
 
+    @secure_communication()
+    def clear_sequence(self):
```

Comment:
Bunch of missing docstrings. By the way docstrings should start with a capital and finish with a blank line.

Reply:
ok